scripts/XLSXHandler.py

```diff
@@ -132,42 +131,43 @@ def __createBlock(self, domain, prefix, block):
             if label in self._labelsMap:
                 printf('Warning: Found a label that has already been assigned to an intent and will be overwritten. Label: %s\n', label)
             del block[0]
-            if not block or not block[0][0]:
-                printf('WARNING: First cell of the goto block does not contain any data. (domain=%s, prefix=%s, label=%s)\n', domain, prefix, label)
+            if not block or not block[0][0]:	
+                printf('WARNING: First cell of the goto block does not contain any data. (domain=%s, prefix=%s, label=%s)\n', domain, prefix, label)	
                 return
             firstCell = block[0][0]
 
         # If it's entity block, load the entity
         if firstCell.startswith(u'@'):
             self.__handleEntityBlock(block)
             return
-
+        
         # Check the intent name
         conditionHasX = Dialog.X_PLACEHOLDER in firstCell
-        intentName = firstCell
+        intent = unidecode.unidecode(firstCell)
 
         if self.__isConditionBlock(firstCell):
             if conditionHasX and block[1][0]:
-                intentName = re.sub(Dialog.X_PLACEHOLDER, block[1][0], firstCell)
+                intent = re.sub(Dialog.X_PLACEHOLDER, block[1][0], firstCell)
         else:
             if firstCell.startswith(u'#'):
-                intentName = firstCell[1:]
+                intent = firstCell[1:] 
             else:
-                # Create intent name from first sentence by replacing all spaces with underscores and removing accents, commas and slashes
-                intentName = re.sub("[/,?']", '', re.sub(' ', '_', unidecode.unidecode(intentName), re.UNICODE))
-
-            # check intent name
-            fullIntentName = toIntentName(NAME_POLICY, None, domain, prefix, intentName)
+                # Create intent name from first sentence by replacing all spaces with underscores and removing accents, commas and slashes	+                # Create fully qualified intent name
+                # intent = re.sub("[/,?']", '', re.sub(' ', '_', unidecode.unidecode(intent), re.UNICODE))
+                # Create fully qualified intent name
+                # intent = unidecode.unidecode(intent)
+                intent = toIntentName(NAME_POLICY, None, domain, prefix, intent)
 
-            self._dialogData.getIntentData(fullIntentName, domain)
-            self._dataBlocks.append((domain, prefix, fullIntentName, block))
-            if label:
-                self._labelsMap[label] = fullIntentName.decode('utf-8')
+        self._dialogData.getIntentData(intent, domain)
+        self._dataBlocks.append((domain, prefix, intent, block))
+        if label:
+            self._labelsMap[label] = intent
```

Comment:
This change fixes <x> blocks missing in dialog xml file.

Reply:
Somehow incorporated in new code.